scripts/rspec/rspec.ps1

```diff
@@ -267,22 +267,36 @@ function AppendVbTestCase($ruleTestsFolder) {
     $existingClassText = Get-Content -Path "${ruleTestsFolder}\\${csClassName}Test.cs" -Raw
     $snippetText = Get-Content -Path "${RuleTemplateFolder}\\VbNetTestSnippet.cs" -Raw
 
-    $namespaceToken = "using CS = SonarAnalyzer.Rules.CSharp;"
+    $usingToken = "using SonarAnalyzer.Rules.CSharp;"
+    $csUsingToken = "using CS = SonarAnalyzer.Rules.CSharp;"
+    $vbNetUsingToken = "using VB = SonarAnalyzer.Rules.VisualBasic;"
+    $namespaceToken = "namespace SonarAnalyzer.UnitTest.Rules"
     $token = "    }"
-    $idx = $existingClassText.LastIndexOf($token)
-    $namespaceTokenIdx = $existingClassText.LastIndexOf($namespaceToken)
-    $newText = ""
+    $usingTokenIdx = $existingClassText.IndexOf($usingToken)
+    $csObjectCreation = "new ${csClassName}"
```

Comment:
Let's remove this variable. It's used only once and it makes the `Replace` hard to read.